pkg/metacni/metacni.go

```diff
@@ -207,14 +210,54 @@ func popDevice(devicePool string, allocatedDevices map[string]*[]string)(string,
   return device, nil
 }
 
+func setIpv6Sysctls(args *cniArgs) error {
+  runtime.LockOSThread()
```

Comment:
regardless of the implementation specifics: Pod NetNS manipulation code we usually put into danmep package, and invoke through an interface
I think this kind of code would also belong there (possibly together with the function "putDanmEp", if you decide to implement my comment

Reply:
code has been refactored heavily...

Reply:
Done.